storage_v2/tracestore/reader.go

```diff
@@ -36,14 +37,18 @@ type Reader interface {
 	// multiple tags) must apply to the same span within a trace, or can be satisfied
 	// by different spans.
 	//
-	// If no matching traces are found, the function returns (nil, nil).
-	FindTraces(ctx context.Context, query TraceQueryParameters) ([]ptrace.Traces, error)
+	// There is no guarantee that all spans for a single trace are returned in a single chunk.
+	// However, there is a guarantees that all spans for a single trace are returned in
+	// one or more consecutive chunks, as if the total output is grouped by trace ID.
```

Comment:
if the output is grouped by traceID, why isn't it returned in a single chunk?

Reply:
same reason why GetTrace doesn't return a single chunk - for performance reason a large trace may need to be split. I'll clarify in the comment.